modules/agent-install-dns-none.adoc

```diff
@@ -0,0 +1,172 @@
+:_content-type: CONCEPT
+[id="agent-install-dns-none_{context}"]
+= Platform "none" DNS requirements
+
+In {product-title} deployments, DNS name resolution is required for the following components:
+
+* The Kubernetes API
+* The {product-title} application wildcard
+* The control plane and compute machines
```

Comment:
Is this really a strict requirement within the current scope? We have, for example, a testing implementation where the external load balancer uses directly the instances IP. cc @rwsu